### DISCUSSION TITLE
Documentation for macOS

### DISCUSSION THREAD

**Discussion by macsrwe** [Kozea/Radicale#1440] [General]
This is a suggestion, not a bug.

I discovered this package today because Apple recommends it as one of the replacements for their own calendar server which they are discontinuing soon. Of the four choices it looked like the best fit for someone who just wants to retain the private SOHO calendar facility they will soon be needing to transport from OS X Server.

In reading the various web pages, whenever I came across a discussion that was platform-specific, the macOS section just said "to be supplied."

There is a certain amount of tick-tock urgency here, especially given that you are being recommended by Apple. Perhaps some effort could be spent on supplying macOS instructions.

Thanks.

**Comment by macsrwe** [comment]
So, 5½ years later, this suggestion is demoted from an issue to a discussion. With this level of responsiveness, I'm sure glad I didn't pursue this. 

It was way easier to dedicate an obsolete Sierra machine to continue to run Mac OS Server.

**Comment by pbiering** [comment]
This project is open source with a very small amount of contributors which have even changed over the time and have itself limited capacity and focus. Current versions are tested against macOS -> feel free to test and in case of any issue *contribute* fixes or documentation extensions.

**Comment by macsrwe** [comment]
Not my skill set, sorry.
